src/common/util/cards.js

```diff
@@ -201,10 +204,15 @@ export function contractKeywords(sentence) {
 //
 
 export function cardsToJson(cards) {
-  cards = cards.map(c => Object.assign({}, c, {schemaVersion: CARD_SCHEMA_VERSION}));
+  cards = cards.map(card => {
+    card = omit(card, ['abilities', 'command']);
```

Comment:
You should probably omit the `id` (in case it is changed and re-uploaded to be the same as another card's `id`. Also there's no reason to keep`timestamp`. I'm worried about people feeding in false data that could somehow break wordbots.